bpf/init.sh

```diff
@@ -435,6 +436,12 @@ if [ "$MODE" == "ipvlan" ]; then
 fi
 bpf_load $HOST_DEV1 "$OPTS" "egress" bpf_netdev.c bpf_host.o from-netdev $CALLS_MAP
 
+if [ "$MODE" = "vxlan" -o "$MODE" = "geneve" ]; then
```

Comment:
I forget, is direct-routing a follow-up PR later on? Do we need a similar trick there too?

(Less certain about if the traffic is from local host to a remote node's pod IP, but surely for post-proxy traffic it needs to go here)

Reply:
The direct routing is up next. The trick appears to be the exact same ;) But I'm not done testing it and didn't want to get into a spot where it broke something if enabled. That said at the moment the golang side doesn't have any stops if a user enables ipsec and direct mode. Currently its a don't do that. The direct routing patch should be posted shortly probably by Monday I hope.